src/viam/examples/modules/complex/CMakeLists.txt

```diff
@@ -154,21 +154,31 @@ install(
 # `test_complex_module.cpp` into the Viam C++ SDK testing suite.
 
 enable_testing()
-target_sources(viamsdk_test
+add_executable(test_complex_module test_complex_module.cpp)
+target_link_libraries(test_complex_module viamsdk_test)
+target_sources(test_complex_module
   PRIVATE
     gizmo/impl.cpp
-    gizmo/impl.hpp
```

Comment:
FYI you can make this same change up above for `complex_module` itself - you don't need to list the header files in the `target_sources`. Same for `complex_module_client`.

Reply:
Good point; done.